.github/workflows/certificate-expiry-check-test-run.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v2
+        uses: aws-actions/configure-aws-credentials@main
```

Comment:
main here but master below, does it work with both values?

Reply:
It does yes but good spot, I'll change it to be consistent